### DISCUSSION TITLE
Why do my docs look different while using the same versions?

### DISCUSSION THREAD

**Discussion by janaSunrise** [wevm/wagmi#226] [Q&A] [answered]
Sorry for this question being off-topic, and not related to this project/repo - Although, this project is amazing, and I love using it! Has helped me a lot.

I noticed that your docs theme is really different than what I have (Mine is the same as the nextra's official docs site), and I'm confused how is it happening. I checked our configs (Almost same), Package versions (The same), and such things. But, I can't help notice it's really different.

Here's an attached screenshot of my docs site:

![Screenshot from 2022-03-01 17-55-03](https://user-images.githubusercontent.com/56873669/156168770-0eb03f08-929e-4211-a334-8da26acdd19a.png)

And, It's really different from the docs site for this app.

For debugging info, Here's info on the project:

Package manager used: `npm`

Package.json:
```json
"dependencies": {
    "next": "12.0.10",
    "nextra": "^2.0.0-alpha.24",
    "nextra-theme-docs": "^2.0.0-alpha.23",
    "react": "17.0.2",
    "react-dom": "17.0.2"
  },
  "devDependencies": {
    "autoprefixer": "^10.4.2",
    "eslint": "8.8.0",
    "eslint-config-next": "12.0.10",
    "postcss": "^8.4.6",
    "prettier": "^2.5.1",
    "tailwindcss": "^3.0.18"
  }
```

Next config:
```js
const withNextra = require('nextra')({
  // Theme
  theme: 'nextra-theme-docs',

  // Config location
  themeConfig: './theme.config.js',

  // Auto image import
  unstable_staticImage: true,

  // Content dump
  unstable_contentDump: true,

  // Stork search
  unstable_stork: false
});

const config = {
  reactStrictMode: true
};

module.exports = withNextra(config);
```

Let me know if anything is needed! Would be amazing if you can look into why the sites are so different - I really love the new layout and design, and that's why I want to get it too.

**Comment by tmm** [comment, marked as answer]
Try removing `.next` and building again. Also, double check that the correct version is installed.

**Reply by janaSunrise** [reply]
Yep. I had the wrong version in package lock. Had to pin it.
